pom.xml

```diff
@@ -44,7 +44,7 @@
         <sonar.organization>axoniq</sonar.organization>
         <sonar.host.url>https://sonarcloud.io</sonar.host.url>
 
-        <axonserver.api.version>2025.1.0</axonserver.api.version>
+        <axonserver.api.version>2025.1.1-SNAPSHOT</axonserver.api.version>
```

Comment:
Oowh, I already said it offline, but I recalled yesterday evening, that if Axon Framework is to use this functionality in AF4, it needs to go into a JDK8 compliant Connector version. Would it be feasible to move the connector changes to the latest 2024.x branch perhaps? :angel: